### ISSUE TITLE
NullPointerException: Attempt to invoke virtual method 'java.lang.String android.content.Context.getString(int)' on a n...

### ISSUE THREAD

**Issue by oguzkocer** [wordpress-mobile/WordPress-Android#12394]
**Sentry Url**: https://sentry.io/share/issue/0049a671b0ab4ed2802cf4a5cfb04207/
**User Count**: 42
**Count**: 46
**First Release**: org.wordpress.android@15.1-rc-1+881
**First Seen**: 2020-06-21T10:05:52.439000Z
**Last Seen**: 2020-07-09T05:47:09Z
**24 Hours**: 4
**30 Days**: 45

```
NullPointerException: Attempt to invoke virtual method 'java.lang.String android.content.Context.getString(int)' on a null object reference
     at org.wordpress.android.ui.prefs.SiteSettingsInterface.loadCachedSettings(SiteSettingsInterface.java:1072)
     at org.wordpress.android.ui.prefs.SiteSettingsInterface.init(SiteSettingsInterface.java:998)
     at org.wordpress.android.ui.prefs.SiteSettingsFragment.lambda$onResume$1(SiteSettingsFragment.java:365)
     at org.wordpress.android.ui.prefs.-$$Lambda$SiteSettingsFragment$zJzgc1xu1BEe9oks5o2qBTk6IcQ.run(null:2)
...
(1 additional frame(s) were not displayed)
```

**Comment by bummytime** [comment]
Updated sentry URL: https://sentry.io/share/issue/749a54cba25644d4a8ff59e1c1c3b3b3/
50 users since Jun 21st — Adding to groundskeeping.

**Comment by designsimply** [comment]
**Events in the last 90d:** 653
**Users affected in the last 90d:** 477
<sub>WORDPRESS-ANDROID-DQZ: https://sentry.io/share/issue/749a54cba25644d4a8ff59e1c1c3b3b3/</sub>

<a href="https://sentry.io/organizations/a8c/discover/results/?display=daily&field=timestamp&field=version&field=os&field=device.family&field=culprit&name=All+Events&project=1438088&query=issue.id%3A1737475229&sort=-timestamp&statsPeriod=90d&widths=-1&widths=-1&widths=-1&widths=-1&widths=-1&yAxis=count%28%29">![image](https://user-images.githubusercontent.com/1119271/96306754-889a9600-0fbd-11eb-8285-4d8e04b93f07.png)</a>

**Comment by designsimply** [comment]
Small increase in events but slight decrease in number of users affected. It doesn't have enough occurrences to send to the Prioritized list in Groundskeeping yet though so it's still on the To Do list for triage again next month.

**Events in the last 90d:** 884
**Users affected in the last 90d:** 438
<sub>WORDPRESS-ANDROID-DQZ: https://sentry.io/share/issue/749a54cba25644d4a8ff59e1c1c3b3b3/</sub>

<a href="https://sentry.io/organizations/a8c/discover/results/?display=daily&field=timestamp&field=version&field=os&field=device.family&field=culprit&name=All+Events&project=1438088&query=issue.id%3A1737475229&sort=-timestamp&statsPeriod=90d&widths=-1&widths=-1&widths=-1&widths=-1&widths=-1&yAxis=count%28%29">![image](https://user-images.githubusercontent.com/1119271/100553323-b163ab80-324a-11eb-80cf-8a087f8ce209.png)</a>

**Comment by designsimply** [comment]
**Events in the last 90d:** 1,000
**Users affected in the last 90d:** 296
<sub>WORDPRESS-ANDROID-DQZ: https://sentry.io/share/issue/749a54cba25644d4a8ff59e1c1c3b3b3/</sub>

<a href="https://sentry.io/organizations/a8c/discover/results/?display=daily&field=timestamp&field=version&field=os&field=device.family&field=culprit&name=All+Events&project=1438088&query=issue.id%3A1737475229&sort=-timestamp&statsPeriod=90d&widths=-1&widths=-1&widths=-1&widths=-1&widths=-1&yAxis=count%28%29">![image](https://user-images.githubusercontent.com/1119271/105411550-c4627e80-5bf0-11eb-8b02-2843556339e1.png)</a>

<a href="https://sentry.io/organizations/a8c/discover/results/?display=daily&field=timestamp&field=version&field=os&field=device.family&field=culprit&name=All+Events&project=1438088&query=issue.id%3A1737475229+version%3A16.3%2A&sort=-timestamp&statsPeriod=90d&widths=-1&widths=-1&widths=-1&widths=-1&widths=-1&yAxis=count%28%29">![image](https://user-images.githubusercontent.com/1119271/105411805-199e9000-5bf1-11eb-8c59-e0a9d25fb1c5.png)</a>

This appears to be fixed as of v16.3. Recommend closing.

**Comment by designsimply** [comment]
Confirmed no new occurrences in 16.3+. Closing as fixed. 🎉
